.README/rules/arrow-parens.md

```diff
@@ -0,0 +1,18 @@
+### `arrow-parens`
+
+_The `--fix` option on the command line automatically fixes problems reported by this rule._
+
+Enforces the consistent use of parentheses in arrow functions.
+
+This rule has a string option and an object one.
```

Comment:
Grabbed all this stuff from the `eslint` documentation [here](https://eslint.org/docs/rules/arrow-parens).